Add support for backup directive for VS and TS (#4653)
nginxinc/kubernetes-ingress: examples/custom-resources/backup-directive/transport-server/README.md

# Support for Backup Directive in Transport Server

NGINX Ingress Controller supports routing requests to a service specified as `Backup`.

> The `backup` service is of type
> [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname).

Reviewer: ```suggestion
`backup` is an [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname) service.
```

> [!NOTE]

> > Support of the services of type
> > [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
> > is only available in NGINX Plus.

Reviewer: ```suggestion
> The [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname) service is only available with NGINX Plus.
```

This phrasing is more concise, and brings the subject (The ExternalName service) to the start of the sentence.

I suggest "with" instead of "in", because "In" suggests that it's NGINX Plus managing the service, not NGINX Ingress Controller.

> In this example, we will use two configurations of the
> [tls-passthrough](/examples/custom-resources/tls-passthrough) example.

Reviewer: ```suggestion
For this example, we will use two [tls-passthrough](/examples/custom-resources/tls-passthrough) configurations.
```
Reduces redundant use of the word "example".

One will be deployed in the `default` namespace, and the other in the `external-ns` namespace.

The application in the `external-ns` namespace will respond to our requests when main application is unavailable.

## Prerequisites

1. Configure the F5 NGINX Ingress Controller deployment with the following flags:

   ```shell
   -enable-custom-resources
   -enable-tls-passthrough
   -watch-namespace=nginx-ingress,default
   ```

   We configure the `-watch-namespace` flag to only watch the `nginx-ingress` and `default` namespaces.

>    This ensures that the F5 NGINX Ingress Controller will treat our service in the `external-ns` namespace

Reviewer: ```suggestion
   This ensures that the NGINX Ingress Controller will treat our service in the `external-ns` namespace
```

   as an external service.

2. Follow the [installation](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)

>    instructions to deploy the F5 NGINX Ingress Controller.

Reviewer: ```suggestion
   instructions to deploy the NGINX Ingress Controller.
```

3. Save the public IP address of the F5 NGINX Ingress Controller into a shell variable:

    ```shell
    IC_IP=XXX.YYY.ZZZ.III
    ```

> 4. Save the HTTPS port of the F5 NGINX Ingress Controller into a shell variable:

Reviewer: ```suggestion
4. Save the HTTPS port of the NGINX Ingress Controller into a shell variable:
```
